### ISSUE TITLE
Firmware evolution 44 breaks integration.

### ISSUE THREAD

**Issue by darkl0rd** [lkempf/casambi-bt-hass#87]
**Describe the bug**
After upgrading to firmware evolution 44, home assistant is unable to create a network and in turn no lights/switches are detected

**To Reproduce**
1. Upgrade firmware to evolution 44.

**Expected behavior**
Everything to still work

**Screenshots**

**Versions**
- HA 2024.9.3
- Integration version: 0.1.2

**Logs**
```
2024-09-25 07:04:25.773 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry xxxx for casambi_bt: Unexpected error creating network 2B:DF:2
2:B0:C2:03
Traceback (most recent call last):
  File "/config/custom_components/casambi_bt/__init__.py", line 103, in connect
    await self.casa.connect(device, self.password)
  File "/usr/local/lib/python3.12/site-packages/CasambiBt/_casambi.py", line 147, in connect
    await self._connectClient()
  File "/usr/local/lib/python3.12/site-packages/CasambiBt/_casambi.py", line 158, in _connectClient
    raise e
  File "/usr/local/lib/python3.12/site-packages/CasambiBt/_casambi.py", line 154, in _connectClient
    await self._casaClient.exchangeKey(self._casaNetwork.getKeyStore())  # type: ignore[union-attr]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/CasambiBt/_client.py", line 158, in exchangeKey
    raise ProtocolError(
CasambiBt.errors.ProtocolError: Unexpected answer from device! Wrong device or protocol version?

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/casambi_bt/__init__.py", line 32, in async_setup_entry
    await api.connect()
  File "/config/custom_components/casambi_bt/__init__.py", line 116, in connect
    raise ConfigEntryError(
homeassistant.exceptions.ConfigEntryError: Unexpected error creating network 2B:DF:22:B0:C2:03
```

**Comment by lkempf** [comment]
This is being tracked as https://github.com/lkempf/casambi-bt/issues/28
Still leaving this open for visibility.

**Comment by Alexei-vk2** [comment]
Hello!

Describe the bug
After upgrading to firmware evolution 44, home assistant is unable to create a network and in turn no lights/switches are detected


2024-09-26 12:19:29.724 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'casambi_bt' accesses hass.helpers.httpx_client. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from httpx_client directly at custom_components/casambi_bt/__init__.py, line 75: hass.helpers.httpx_client.get_async_client(), get_cache_dir(hass), please report it to the author of the 'casambi_bt' custom integration
2024-09-26 12:19:29.755 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to read_text with args (PosixPath('/config/.storage/casambi_bt/.cachever'),) inside the event loop by custom integration 'casambi_bt' at custom_components/casambi_bt/__init__.py, line 102: await self.casa.connect(device, self.password) (offender: /usr/local/lib/python3.12/site-packages/CasambiBt/_cache.py, line 38: cacheVer = int(self._cacheVersionFile.read_text())), please report it to the author of the 'casambi_bt' custom integration
For developers, please see https://developers.home-assistant.io/docs/asyncio_blocking_operations/#read_text
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module>
    sys.exit(main())
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main
    exit_code = runner.run(runtime_conf)
  File "/usr/src/homeassistant/homeassistant/runner.py", line 189, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 674, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 641, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 1990, in _run_once
    handle._run()
  File "/usr/local/lib/python3.12/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 165, in async_setup_component
    result = await _async_setup_component(hass, domain, config)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 461, in _async_setup_component
    await asyncio.gather(
  File "/usr/src/homeassistant/homeassistant/setup.py", line 463, in <genexpr>
    create_eager_task(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 45, in create_eager_task
    return Task(coro, loop=loop, name=name, eager_start=True)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 752, in async_setup_locked
    await self.async_setup(hass, integration=integration)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/casambi_bt/__init__.py", line 31, in async_setup_entry
    await api.connect()
  File "/config/custom_components/casambi_bt/__init__.py", line 102, in connect
    await self.casa.connect(device, self.password)

**Comment by Alexei-vk2** [comment]
how to fix the problem?

**Comment by lkempf** [comment]
This should be fixed as of v0.1.3.
